sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -20,10 +20,14 @@ class SdsInABoxStack(Stack):
 
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
+        
+ ########### INIT
+        # Determines the initial configuration
+        SDS_ID = self.node.try_get_context("SDSID")
```

Comment:
```suggestion
```
We've already parsed it in "app.py", so should we pass it into this function rather than trying to get it again?

Also, if it is None I don't think this is working because there is no fallback like in the original.

Reply:
I initially had it passed in as variables to the function, but I think there was a reason that I stopped passing it in and just looking at the context again.  I'd need to play around with it again to determine why that way.

Reply:
:+1: The reason to pass it in would be that currently if I set my ID to "random", then that will put the random string here instead of the randomly generated string from the app.